packages/calypso-products/src/is-titan-mail.ts

```diff
@@ -1,7 +1,11 @@
 import { camelOrSnakeSlug } from './camel-or-snake-slug';
-import { TITAN_MAIL_SLUGS } from './constants';
+import { TITAN_MAIL_MONTHLY_SLUG, TITAN_MAIL_SLUGS } from './constants';
 import type { WithCamelCaseSlug, WithSnakeCaseSlug } from './types';
 
 export function isTitanMail( product: WithCamelCaseSlug | WithSnakeCaseSlug ): boolean {
 	return ( TITAN_MAIL_SLUGS as ReadonlyArray< string > ).includes( camelOrSnakeSlug( product ) );
 }
+
+export function isTitanMailMonthly( product: WithCamelCaseSlug | WithSnakeCaseSlug ): boolean {
```

Comment:
I think it may make more sense to push this new function into a dedicated `is-titan-mail-monthly.ts` file. That way our bundler can include only the code it needs for each case, and not both functions even when only one is being used.

Reply:
👍